lib/src/easy_date_time_formatting.dart

```diff
@@ -310,6 +310,9 @@ extension EasyDateTimeFormatting on EasyDateTime {
 /// }
 /// ```
 class EasyDateTimeFormatter {
+  /// Cache all created formatters, as they are immutable.
+  static Map<String, EasyDateTimeFormatter> _availablePatterns = {};
```

Comment:
You might consider making this static final to avoid accidental reassignment.

Reply:
fixed.   thanks for the catch.